### PR TITLE
update shentu info & add shentu testnet

### PR DESCRIPTION
**Pull request by 0311xuyang**
Changelog:

- update shentu website
- update shentu api servers
- update shentu binary versions
- add shentu testnet

### CODE REVIEW

File: shentu/chain.json [hunk 1]
```diff
@@ -36,18 +36,18 @@
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/ShentuChain/mainnet/main/shentu-2.2/genesis.json"
     },
-    "recommended_version": "v2.8.0",
```
Comment (JeremyParish69) [inline shentu/chain.json:39]: Please retain v2.8.0 inside the versions array

File: testnets/shentutestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "name": "Shentu",
+      "display": "ctk",
+      "symbol": "CTK",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png",
```
Comment (JeremyParish69) [inline testnets/shentutestnet/assetlist.json:27]: Please use image_sync, pointing back to mainnet ctk. Something like:
```
"image_sync": {
  "chain_name": "shentu",
  "base_denom": "uctk"
},
```

File: testnets/shentutestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  },
+  "explorers": [
+    {
+      "kind": "Shentu Explorer",
+      "url": "https://explorer.shentu.org/?chain=yulei-2.1",
+      "tx_page": "https://explorer.shentu.org/transactions/${txHash}?chain=yulei-2.1"
+    }
+  ],
+  "images": [
+    {
```
Comment (JeremyParish69) [inline testnets/shentutestnet/chain.json:222]: Please use image sync, pointing to the asset. Something ike:
`"image_sync": {
  "chain_name": "shentutestnet",
  "base_denom": "uctk"
},`

File: shentu/chain.json (hunk 1 above)
Comment (0311xuyang) [reply shentu/chain.json:39]: Added all history versions in versions array field

File: testnets/shentutestnet/assetlist.json (hunk 2 above)
Comment (0311xuyang) [reply testnets/shentutestnet/assetlist.json:31]: added the image_sync field

File: testnets/shentutestnet/chain.json (hunk 3 above)
Comment (0311xuyang) [reply testnets/shentutestnet/chain.json:222]: added the image_sync field

File: testnets/shentutestnet/chain.json [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      }
+    ],
+    "ibc": {
+      "type": "go",
+      "version": "v4.2.1",
```
Comment (JeremyParish69) [inline testnets/shentutestnet/chain.json:158]: This version number must match above (should be v7.4.0).
I think this is the last issue.

Comment (0311xuyang) [reply testnets/shentutestnet/chain.json:158]: Ah yes, forgot to update the testnet, sry ..

**Comment by 0311xuyang** [comment]
Hi the following commit merged into another PR https://github.com/cosmos/chain-registry/pull/5389

**Pull request closed without merging**